CONTRIBUTING.md

```diff
@@ -189,16 +189,23 @@ Icon metadata should be added to the `_data/simple-icons.json` file. Each icon i
   * A `hex` color value that matches the brand's primary color. All uppercase and without the `#` pound symbol.)
   * The `source` URL of the logo being used. There are [more details below](#source-guidelines).
 
-There is also an optional field that may be used to specify the brand guidelines/presskit/etc. This is useful if the SVG file was sourced from a different place.
+Additionally, there are also optional fields that may provided for an icon:
```

Comment:
I wonder should we create a new section for adding icon data an link to it from here. The new section would cover all the optional entries such as guidelines, license, the upcoming aliases plus any others we add in the future.

My reason for suggesting it is twofold:
1. We avoid overwhelming new contributors as we add more entries to the JSON - yes, when adding icons, the more data that is included the better but as long as the bare minimum of `title`, `hex`, `source` and, where applicable, `slug` are included then we're off to a good start, and,
2. The addition of all this new data for existing icons is going to be separate to the addition of new icons and I'm hoping that with a dedicated section in the guidelines it might encourage community members to start helping us out in adding all the data we need.

Reply:
Good idea, I'll draft something of that nature :+1:

Reply:
Yeah, it can wait; it's only really going to become an issue when we add the aliases.

Reply:
See https://github.com/simple-icons/simple-icons/pull/5068 :+1: